learn/guides/calling-java-code-from-ballerina.md

```diff
@@ -25,11 +25,11 @@ redirect_from:
 ---
 
 ## Writing Ballerina Bindings
-The `bindgen` is a CLI tool, which eases the process of generating Ballerina bindings for given Java APIs. This guide shows you how to use it. For more details on how the `bindgen` tool works, see [The Bindgen Tool](/learn/java-interoperability/the-bindgen-tool/).
+The `bindgen` is a CLI tool, which eases the process of generating Ballerina bindings for given Java APIs. This guide shows you how to use it. For more details on how the `bindgen` tool works, see [The Bindgen Tool](/learn/java-interoperability/the-bindgen-tool/) reference guide.
```

Comment:
```suggestion
The `bindgen` is a CLI tool, which eases the process of generating Ballerina bindings for given Java APIs. This guide shows you how to use it. For more details on how the `bindgen` tool works, see [The Bindgen Tool](/learn/java-interoperability/the-bindgen-tool/).
```
Do not need to say `reference guide` again explicitly. Please refer the guidelines on how to give reference links.